SPECS/openssl.spec

```diff
@@ -29,7 +29,7 @@ print(string.sub(hash, 0, 16))
 Summary: Utilities from the general purpose cryptography library with TLS implementation
 Name: openssl
 Version: 3.0.7
-Release: 27%{?dist}.0.2.2
```

Comment:
I know this ship is well sailed, but I'm concerned about bumping at the end here as our dist tags are not very compatible with RHEL/Rocky's. For example, Rocky's dist tag of elMajor[_Minor] in this version, creating 3.0.7-27.el8_10.0.2.2 (for example) will always precede 3.0.7-27.el8_10.ciqfips.0.2.3, because of the .ciqfips being compared against .0.